source/libtomo/prep/CMakeLists.txt

```
@@ -1,31 +1,24 @@
-# Locate sources for this project headers are included for IDEs
-file(GLOB prep_tomo_sources
-    ${CMAKE_CURRENT_LIST_DIR}/*.c)
+set(HEADERS "${tomopy_SOURCE_DIR}/include/libtomo/prep.h"
+            "${tomopy_SOURCE_DIR}/include/libtomo/stripe.h")
 
-# core sources
-set(PREP_LIBRARY_SOURCES ${prep_tomo_sources})
+add_library(tomo-prep SHARED prep.c stripe.c ${HEADERS})
 
-# create library
-add_library(libtomopy-prep SHARED ${PREP_LIBRARY_SOURCES})
+target_include_directories(
+  tomo-prep
+  PUBLIC $<BUILD_INTERFACE:${tomopy_SOURCE_DIR}/include/libtomo>
+         $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include/libtomo>
+  PRIVATE ${CMAKE_CURRENT_LIST_DIR})
 
-list(APPEND TARGET_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/../../include/prep")
+target_compile_definitions(tomo-prep PUBLIC ${${PROJECT_NAME}_DEFINITIONS})
```

```suggestion
target_compile_definitions(tomo-prep PRIVATE ${${PROJECT_NAME}_DEFINITIONS})
```

Same sort of note as before. Downstream projects probably don't need these definitions.